src/arviz_plots/plots/lm_plot.py

```diff
@@ -1,91 +1,95 @@
     get_group,
+    get_visual_kwargs,
     process_group_variables_coords,
     set_wrap_layout,
 )
-from arviz_plots.visuals import fill_between_y, labelled_x, labelled_y, line_xy, scatter_xy
+from arviz_plots.visuals import (
+    ci_line_y,
+    fill_between_y,
+    labelled_x,
+    labelled_y,
+    line_xy,
+    scatter_xy,
+)
 
 
 def plot_lm(
     dt,
-    y=None,
     x=None,
-    y_pred=None,
-    x_pred=None,
+    y=None,
+    y_obs=None,
+    smooth=True,
     filter_vars=None,
     group="posterior_predictive",
     coords=None,
     sample_dims=None,
     ci_kind=None,
     ci_prob=None,
-    line_kind=None,
+    point_estimate=None,
     plot_collection=None,
     backend=None,
     labeller=None,
     aes_by_visuals: Mapping[
         Literal[
-            "ci_line",
-            "central_line",
-            "ci_fill",
-            "scatter",
+            "pe_line",
+            "ci_band",
+            "ci_bounds",
+            "ci_line_y",
+            "observed_scatter",
             "xlabel",
             "ylabel",
         ],
-        Sequence[str],
+        list[str],
     ] = None,
     visuals: Mapping[
         Literal[
-            "ci_line",
-            "central_line",
-            "ci_fill",
-            "scatter",
+            "pe_line",
+            "ci_band",
+            "ci_bounds",
+            "ci_line_y",
+            "observed_scatter",
             "xlabel",
             "ylabel",
         ],
         Mapping[str, Any] | Literal[False],
     ] = None,
     stats: Mapping[
-        Literal["credible_interval", "point_estimate"],
+        Literal["credible_interval", "point_estimate", "smooth"],
         Mapping[str, Any] | xr.Dataset,
     ] = None,
     **pc_kwargs,
 ):
-    """Posterior predictive and mean plots for regression-like data..
+    """Posterior predictive and mean plots for regression-like data.
 
     Parameters
     ----------
     dt : DataTree
         Input data
-    y : str or DataArray, optional
-        Target variable. If None (default), the first variable in "observed_data" is used.
-    x : str or list of str or DataArray or Dataset, optional
-        Independent variable(s). If None (default), all variables in "constant_data" are used.
-    y_pred : str or DataArray, optional
-        Predicted values.
-        If None (default), the variable in the specified group with the same name as y is used.
-    x_pred : str or list of str or DataArray or Dataset, optional
-        Independent variable(s) for predictions.
-
-        If None (default), and if group is "predictions", all variables corresponding to x data
-        in "predictions_constant_data" group are used. If group is "posterior_predictive",
-        x is used.
+    x : str  optional
+        Independent variable. If None, use the first variable in constant_data group.
+    y : str optional
+        Response variable or linear term. If None, use the first variable in observed_data group.
+    y_obs : str or DataArray, optional
```

Comment:
I think it would be good to explain the difference between `y` and `y_obs` a bit more.  They're looked up in different groups, right? I would mention it. 

Also, why does it make different assumptions about `y` and `y_obs`? I can imagine it's related to variables available in the different groups, but I'm not sure.